spec/controllers/restful_redirect_controller_spec.rb

```diff
@@ -44,12 +44,6 @@
       expect(response).to redirect_to(:controller => "vm_cloud", :action => "show", :id => vm_cloud.id)
     end
 
-    it 'redirects to provider_foreman controller' do
```

Comment:
did we lose this test for for configuration managers? I don't see elsewhere in this code, but maybe I just missed it.

Reply:
good catch @Fryguy that test was removed because i removed https://github.com/ManageIQ/manageiq-ui-classic/pull/6782/files#diff-54afc0af5eff6d605e58aea868f5db92L9-L10 from controller but it appears we do no need that code, so i will adjust and put the test and controller code back in